src/backend/config.ts

```diff
@@ -133,6 +137,14 @@ export const oboTokenConfig: ITokenRequest = {
     tokenUri: nodeConfig.tokenURI,
 };
 
+export const msGraphOBOTokenConfig: ITokenRequest = {
```

Comment:
You dont need this when configured in the package.

Reply:
ok